.storybook/theme.js

```diff
@@ -0,0 +1,40 @@
+import { create } from "@storybook/theming";
+import logo from "./static/logo.svg";
```

Comment:
did you need your webpack changes to support this?

Reply:
I am thinking not (?) I am perplexed by this compared to my other branch where I had lots of logo rendering troubleshooting that did involve the webpack changes. I tried running this before adding any of that and the logo renders 🤷‍♀️